app/views/datasets/show.html.erb

```diff
@@ -41,7 +41,7 @@
 
   <%= render "additional_info" %>
 
-  <% unless no_documents?(@dataset.datafiles) %>
+  <% unless @dataset.docs.empty? %>
```

Comment:
We'll need to make sure `@dataset.docs` returns `[]` here. :)

https://sentry.io/government-digital-service/find/issues/437118602/

Reply:
It does return [ ] this error was because the ES mappings hadn't been updated on Publish. I opened a PR to fix. https://github.com/alphagov/datagovuk_publish/pull/500